src/fft.rg

```diff
@@ -13,74 +13,135 @@
 -- limitations under the License.
 
 --[[--
-Regent FFT library.
+Regent FFT library
```

Comment:
I think this is a doc comment format (thus the `--[[--` weird start to this comment) so the trailing `.` is not a mistake here.

Reply:
Added the period back.